gpt4all-backend/sysinfo.h

```diff
@@ -41,6 +43,10 @@ static long long getSystemTotalRAMInBytes()
     memoryStatus.dwLength = sizeof(memoryStatus);
     GlobalMemoryStatusEx(&memoryStatus);
     totalRAM = memoryStatus.ullTotalPhys;
+#elif defined(__FreeBSD__)
+    int mib[2] = {CTL_HW, HW_PHYSMEM};
```

Comment:
This is fine with me, but I have no way of testing it. If you verify this works for you I'm happy to add it, but you have to remove the draft, verify it works, and remove the maintenance tool change.